data-avro/src/test/java/com/linkedin/data/avro/TestSchemaTranslator.java

```diff
@@ -1660,6 +1660,46 @@ public void testToAvroSchema_testTypeRefAnnotationPropagation(String schemaBefor
               null,
               null,
               null
+          },
```

Comment:
these are not related to your change.. please check

Reply:
yes updated the branch